zkbreacharbiter_test.go

```diff
@@ -171,6 +171,45 @@ func initTestMerchState(DBPath string, skM string, payoutSkM string, disputeSkM
 	return err
 }
 
+func setupMerchChannelState(escrowTxid string, DBPath string, newStatus string) error {
+	zkMerchDB, err := zkchanneldb.OpenMerchBucket(DBPath)
+	if err != nil {
+		zkchLog.Error(err)
+		return err
+	}
+
+	merchState, err := zkchanneldb.GetMerchState(zkMerchDB)
+	if err != nil {
+		zkchLog.Error(err)
+		return err
+	}
+
+	// Flip escrowTxid to Big Endian to match how it is stored in
+	// merchState.ChannelStatusMap
+	//This works because hex strings are of even size
+	s := ""
+	for i := 0; i < len(escrowTxid)/2; i++ {
```

Comment:
```
for i := 0; i < len(escrowTxid); i += 2 {
    s = escrowTxid[i:i+2] + s
}
```

Reply:
nice. All cases of this have changed now